### ISSUE TITLE
Keyboard focus order is not logical under SSMA Feedback window: A11y_WinForm.NETRuntime_FocusOrder.

### ISSUE THREAD

**Issue by shindepra** [dotnet/winforms#13757]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."

### GitHub Tags:
#A11ySev2; #A11yTCS; #WCAG2.4.3; #Win32; #DesktopApp; #Win11; #WinForms .NET Runtime; #BM_WinForms.NETRuntime_Win32_June2025; #FocusOrder; #A11yMAS

### Environment Details:
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061  

### Repro Steps:

1. Open SQL Server Migration Assistant for oracle.  
2. Splash screen appears and SSMA is launched 
3. Navigate to connect to oracle and connect to target button and connect it.  
4. Navigate to the 'Tools' menu and select it.  
5. Navigate to 'Convert schema' menu item and select it.  
6. Navigate to the SSMA Feedback window 
7. Verify whether focus order is logical or not

### Actual Result:
Keyboard focus order is not logical under SSMA Feedback window. 

### Expected Result:
Keyboard focus order should be logical under SSMA Feedback window. 

### User Empact:
Users who rely on the keyboard will be affected if the focus order is not in the visual sequential order. Keyboard users will be affected as they get confused if the focus order is not as expected. 

Sample App used:-

[RadioButtonFocusDemo.zip](https://github.com/user-attachments/files/21483118/RadioButtonFocusDemo.zip)

### Attachment:

<img width="896" height="488" alt="Image" src="https://github.com/user-attachments/assets/085d6ce7-873f-4ef4-8cb5-fade0cfd7ab2" />

https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-kkumar2_microsoft_com/EdG6SWTqMBBNnNMZSHHgupkB1EElv3wCOiOsxfmGGj1XFg?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJPbmVEcml2ZUZvckJ1c2luZXNzIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXciLCJyZWZlcnJhbFZpZXciOiJNeUZpbGVzTGlua0NvcHkifX0&e=SZ6gYK

**Comment by LeafShi1** [comment]
This should be an issue with the design of the SQL Server product. The jump order of the Tab key is based on the `TabIndex` when it is designed.

<img width="2103" height="1469" alt="Image" src="https://github.com/user-attachments/assets/abddf5e3-193f-43a6-9e42-df241fdcc274" />

**Comment by Kirank915** [comment]
@LeafShi1 SSMA team has already fixed the logical order (jump order) of the radio buttons only issue that is not fixed is radio group not getting focus when no radio button is selected.

Could you please check?

**Comment by LeafShi1** [comment]
The GroupBox is a container control that does not respond to keyboard focus. And each RadioButton is an independent control and you cannot set focus directly to a group.
